admin_manual/installation/apps_management_installation.rst

```diff
@@ -71,17 +71,60 @@ The configuration example below shows how to add a second directory, called ``ap
    :emphasize-lines: 9-13
 
 After you add a new directory configuration, you can then move apps from the original app directory to the new one. 
-To do so, follow these steps:
+If you want to identify as an example all non core / non shipped apps, use the following command:
+
+::
+
+  sudo -uwww-data ./occ app:list --shipped=false
+
+Follow these steps to manually move apps:
 
 #. :ref:`Enable maintenance mode <maintenance_commands_label>`.
 #. :ref:`Disable the apps <apps_commands_label>` that you want to move.
-#. Create a new apps directory and assign it the same user and group, and ownership permissions as the core apps directory.
+#. Create a new apps directory like ``apps-external`` and assign it the same user, group and ownership permissions as the core apps directory.
 #. Move the apps from the old apps directory to the new apps directory.
-#. Add a new app directory in ``config/config.php``.
+#. Add the key for the new app directory in ``config/config.php``.
+#. If you're using a cache, such as `Redis <../configuration/server/caching_configuration.html#clearing-the-redis-cache>`_ or `Memcached <../configuration/server/caching_configuration.html#clearing-the-memcached-cache>`_, ensure that you clear the cache.
```

Comment:
The docs convention is to store page links at the bottom of the page, as it makes editing easier. For example:

#. If you're using a cache, such as `Redis`_ or `Memcached`_, ensure that you clear the cache.

... rest of the document...

.. Links

.. _Redis: ../configuration/server/caching_configuration.html#clearing-the-redis-cache
.. _Memcached: ../configuration/server/caching_configuration.html#clearing-the-memcached-cache